### PR TITLE
PREL-260 Enable pmm3-client repo

### PR DESCRIPTION
**Pull request by ademidoff**

### CODE REVIEW

File: scripts/percona-release.sh [hunk 1]
```diff
@@ -51,12 +51,12 @@ function get_repos_from_site {
...
   if [ -z "$REPOSITORIES" ]; then
-    REPOSITORIES="original ps-56 ps-57 ps-80 pxb-24 pxb-80 pxc-56 pxc-57 pxc-80 psmdb-36 psmdb-40 psmdb-42 tools ppg-11 ppg-11.5 ppg-11.6 ppg-11.7 ppg-11.8 ppg-12 ppg-12.2 ppg-12.3 pdmdb-4.2 pdmdb-4.2.6 pdmdb-4.2.7 pdmdb-4.2.8 pdps-8.0.19 pdpxc-8.0.19 pdps-8.0.20 pdps-8.0 pdpxc-8.0 prel proxysql sysbench pt mysql-shell pbm pmm-client pmm2-client pdmdb-4.4 pdmdb-4.4.0 psmdb-44"
+    REPOSITORIES="original ps-56 ps-57 ps-80 pxb-24 pxb-80 pxc-56 pxc-57 pxc-80 psmdb-36 psmdb-40 psmdb-42 tools ppg-11 ppg-11.5 ppg-11.6 ppg-11.7 ppg-11.8 ppg-12 ppg-12.2 ppg-12.3 pdmdb-4.2 pdmdb-4.2.6 pdmdb-4.2.7 pdmdb-4.2.8 pdps-8.0.19 pdpxc-8.0.19 pdps-8.0.20 pdps-8.0 pdpxc-8.0 prel proxysql sysbench pt mysql-shell pbm pmm-client pmm2-client pmm3-client pdmdb-4.4 pdmdb-4.4.0 psmdb-44"
   fi
   REPOSITORIES="${REPOSITORIES/percona/original}"
   for repo in ${REPOSITORIES[@]}
     do
-      if [ ${repo} != "mysql-shell" -a ${repo} != "pmm-client" -a ${repo} != "pmm2-client" -a ${repo} != "pmm2-components" ]; then
+      if ! [[ ${REPO_NAME} =~ mysql-shell|pmm-client|pmm2-client|pmm3-client|pmm2-components ]]; then
```
Comment (ademidoff) [inline scripts/percona-release.sh:59]: This does pretty much the same thing but is however more readable :)

**Review by BupycHuk — APPROVED** [review]

**Comment by ademidoff** [comment]
@EvgeniyPatlan Could you please review it as well?

File: scripts/percona-release.sh [hunk 2]
```diff
@@ -51,12 +51,12 @@ function get_repos_from_site {
...
   if [ -z "$REPOSITORIES" ]; then
-    REPOSITORIES="original ps-56 ps-57 ps-80 pxb-24 pxb-80 pxc-56 pxc-57 pxc-80 psmdb-36 psmdb-40 psmdb-42 tools ppg-11 ppg-11.5 ppg-11.6 ppg-11.7 ppg-11.8 ppg-12 ppg-12.2 ppg-12.3 pdmdb-4.2 pdmdb-4.2.6 pdmdb-4.2.7 pdmdb-4.2.8 pdps-8.0.19 pdpxc-8.0.19 pdps-8.0.20 pdps-8.0 pdpxc-8.0 prel proxysql sysbench pt mysql-shell pbm pmm-client pmm2-client pdmdb-4.4 pdmdb-4.4.0 psmdb-44"
+    REPOSITORIES="original ps-56 ps-57 ps-80 pxb-24 pxb-80 pxc-56 pxc-57 pxc-80 psmdb-36 psmdb-40 psmdb-42 tools ppg-11 ppg-11.5 ppg-11.6 ppg-11.7 ppg-11.8 ppg-12 ppg-12.2 ppg-12.3 pdmdb-4.2 pdmdb-4.2.6 pdmdb-4.2.7 pdmdb-4.2.8 pdps-8.0.19 pdpxc-8.0.19 pdps-8.0.20 pdps-8.0 pdpxc-8.0 prel proxysql sysbench pt mysql-shell pbm pmm-client pmm2-client pmm3-client pdmdb-4.4 pdmdb-4.4.0 psmdb-44"
   fi
   REPOSITORIES="${REPOSITORIES/percona/original}"
   for repo in ${REPOSITORIES[@]}
     do
-      if [ ${repo} != "mysql-shell" -a ${repo} != "pmm-client" -a ${repo} != "pmm2-client" -a ${repo} != "pmm2-components" ]; then
+      if ! [[ ${repo} =~ mysql-shell|pmm-client|pmm2-client|pmm3-client|pmm2-components ]]; then
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:59]: in this case if ! [[ ... ]] it is not very readable so better to invert logic
like this:

if [[ ${repo} =~ mysql-shell|pmm-client|pmm2-client|pmm3-client|pmm2-components ]]; then
  ALIASES+="${repo} "
else
  ALIASES+="${repo//-/} "
fi

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: scripts/percona-release.sh (hunk 2 above)
Comment (ademidoff) [reply scripts/percona-release.sh]: Sure, done!

File: scripts/percona-release.sh [hunk 3]
```diff
@@ -136,6 +137,7 @@ PDPXC80_DESC="Percona Distribution for MySQL 8.0 - PXC"
 PPG_DESC="Percona Distribution for PostgreSQL"
 PDMDB_DESC="Percona Distribution for MongoDB"
 PDPS_DESC="Percona Distribution for MySQL - PS"
+PS_DESC="Percona Server for MySQL - PS"
```
Comment (EvgeniyPatlan) [inline scripts/percona-release.sh:140]: we are not using PS_DESC as we have PS80_DESC

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: scripts/percona-release.sh (hunk 3 above)
Comment (ademidoff) [reply scripts/percona-release.sh:140]: I don't know how, but it seems like I pulled those changes from the main branch (the source [commit](https://github.com/percona/percona-repositories/pull/118/commits/473d6698a3375153fca9505b6ab2a064268ad564)). You may ask @maqeel75 about that one.

Comment (ademidoff) [reply scripts/percona-release.sh]: It's rebased now and free from foreign commits.

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**